chain/chain/src/error.rs

```diff
@@ -128,9 +128,6 @@ pub enum ErrorKind {
     /// Invalid Gas Used
     #[fail(display = "Invalid Gas Used")]
     InvalidGasUsed,
-    /// Invalid Rent Paid
-    #[fail(display = "Invalid Rent Paid")]
-    InvalidRent,
```

Comment:
FYI @kcole16 @vgrichina  we need to update our nearlib here https://github.com/near/near-api-js/blob/master/src.ts/res/error_messages.json#L39 and other places also add `Deprecated` error https://github.com/nearprotocol/nearcore/pull/2258/files#diff-b91de36721d2524f2c32551ade6a05b8R137

Reply:
@ilblackdragon @nearmax please make a near-api-js PR, I'll review. I think it's best if protocol changes are plumbed end to end as a whole by whoever has most context, unless it's too complex to do without platform expertise.

Reply:
FYI @frol , we need RPC regression tests for situations like this.

Reply:
@vgrichina hm, is this not it https://github.com/near/near-api-js/pull/268 ?

Reply:
@ilblackdragon all good then! I just didn't have it in my review queue and it's not linked in PR description.

Reply:
Should we introduce OWNERS file in near-api-js to among other things address this problem?
I just added a PR as external contributor to near-api-js - and no one is assigned for it.

Reply:
Yeah, makes sense for external contributors. But for this particular issue PRs just need to be cross linked IMO.